src/charts/heatmap/heatmap.directive.js

```diff
@@ -271,8 +286,20 @@ angular.module('patternfly.charts').directive('pfHeatmap', function ($compile, $
 
         //Adding events
         blocks.on('mouseover', function () {
+          var fillColor;
           blocks.call(highlightBlock, false);
-          d3.select(this).call(highlightBlock, true);
+          if (scope.maxSize < 16) {
```

Comment:
Thanks for clarifying. I think what I really want to test is blockSize - scope.padding (i.e., the fillSize var). It wasn't correct having to set max-block-size="13" in order for blockSize to be less than 16 and highlighting finally becomes enabled.